### PR TITLE
W-10915394 / Giving Summary Design

### PR DESCRIPTION
**Pull request by maurimart**
- Change the layout of the giving summary component to display the fields in mobile and use slds where possible

# Critical Changes

### CODE REVIEW

**Comment by maurimart** [comment]
[W-10915394](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000tsHViYAM/view)

**Review by JoseRossiSF — APPROVED** [review]

File: force-app/main/default/lwc/givingSummary/givingSummary.html [hunk 1]
```diff
@@ -16,7 +16,10 @@ <h2 class={summaryTitleClass}>
       </template>
       <template if:false={errorMessage}>
         <lightning-layout-item
-          size="4"
+          size="12"
+          small-device-size="12"
```
Comment (gerardorodriguezsf) [inline force-app/main/default/lwc/givingSummary/givingSummary.html:20]: small-device-size="12" and large-device-size="4" can be omitted. Same on the other lightning layouts

**Review by gerardorodriguezsf — APPROVED** [review]
LGTM

File: force-app/main/default/lwc/givingSummary/givingSummary.js [hunk 2]
```diff
@@ -77,19 +77,19 @@ export default class GivingSummary extends LightningElement {
...
+        return 'slds-text-heading_small  slds-text-align_left slds-var-p-around_small';
     }
 
     /**
      * @description returns the classes of the last row if it is mobile or desktop
      */
      get usdValue() {
         if (this.isMobile) {
-            return 'columns-mobile';
+            return 'slds-text-heading_small bold-title';
```
[the remark this answers is not in the document]
Comment (maurimart) [reply force-app/main/default/lwc/givingSummary/givingSummary.js]: good suggestion

File: force-app/main/default/lwc/givingSummary/givingSummary.js [hunk 3]
```diff
@@ -77,19 +77,19 @@ export default class GivingSummary extends LightningElement {
      */
     get lastElementClasses() {
         if (this.isMobile) {
-            return 'columns-mobile slds-var-p-left_small slds-var-p-right_small slds-text-align_left to-uppercase';
+            return 'slds-text-heading_small  slds-var-p-around_small slds-text-align_left';
```
[the remark this answers is not in the document]
Comment (maurimart) [reply force-app/main/default/lwc/givingSummary/givingSummary.js:80]: in this case different paddings are being handled

**Review by lparrott — APPROVED** [review]

**Review by bdvorachek — APPROVED** [review]

**Pull request merged**